<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewEmployee</class>
 <widget class="QWidget" name="AddNewEmployee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddNewEmployee</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>First Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ID number:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtFirstName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtLastName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtIdNum">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Position: </string>
   </property>
  </widget>
  <widget class="QComboBox" name="position_comboBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="submitBtn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>256</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteBtn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Employee</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>submitBtn</sender>
   <signal>clicked()</signal>
   <receiver>AddNewEmployee</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>position_comboBox</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>AddNewEmployee</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteBtn</sender>
   <signal>clicked()</signal>
   <receiver>AddNewEmployee</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>AddNewEmployee</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
